editor/plugins/editor_plugin.cpp

```diff
@@ -428,6 +429,16 @@ void EditorPlugin::remove_export_plugin(const Ref<EditorExportPlugin> &p_exporte
 	EditorExport::get_singleton()->remove_export_plugin(p_exporter);
 }
 
+void EditorPlugin::add_export_platform(const Ref<EditorExportPlatform> &p_platform) {
+	ERR_FAIL_COND(!p_platform.is_valid());
```

Comment:
```suggestion
	ERR_FAIL_COND(p_platform.is_null());
```

Reply:
We could also use `ERR_FAIL_NULL(p_platform)`.

Reply:
Not something done with `Ref` normally, so I wouldn't